include/tune_quda.h

```diff
@@ -204,14 +209,16 @@ namespace quda {
     }
 
     /**
-     * @brief This can't be correctly queried in CUDA for all
-     * architectures so here we set set this.  Based on Table 14 of
-     * the CUDA Programming Guide 10.0 (Technical Specifications per
-     * Compute Capability).
+     * @brief Returns the maximum dynamics shared memory per block.
```

Comment:
Same comment as above

Reply:
That was intentional, but I've restored the reference in a new comment in the else path